testdata/extern_modules/extern-checksum-ebpf.c

```diff
@@ -0,0 +1,37 @@
+#include  "test.h"
+
+/*
+ * This file implements sample C extern function. It contains definition of the following C extern function:
+ *
+ * bool verify_ipv4_checksum(in IPv4_h iphdr)
+ *
+ */
+
+
+/**
+ * This function implements method to verify IP checksum. It shows how to write C extern function for P4 program.
```

Comment:
the second phrase is not necessary.
This function is very useful as it is.

Reply:
Done.